deployment/amplify/production/amplify/#current-cloud-backend/auth/echolocatorCognitoProduction/parameters.json

```diff
@@ -0,0 +1,66 @@
+{
+    "identityPoolName": "echolocatorIdentityPoolProduction",
+    "allowUnauthenticatedIdentities": false,
+    "thirdPartyAuth": false,
+    "lambdaLogPolicy": "echolocatorCognitoProduction_lambda_log_policy",
+    "openIdLambdaRoleName": "echolocatorCognitoProduction_openid_lambda_role",
+    "openIdRolePolicy": "echolocatorCognitoProduction_openid_pass_role_policy",
+    "openIdLambdaIAMPolicy": "echolocatorCognitoProduction_openid_lambda_iam_policy",
+    "openIdLogPolicy": "echolocatorCognitoProduction_openid_lambda_log_policy",
+    "userPoolName": "echolocatorUserPoolProduction",
+    "autoVerifiedAttributes": [
+        "email"
+    ],
+    "mfaConfiguration": "OFF",
+    "mfaTypes": [
+        "SMS Text Message"
+    ],
+    "roleName": "echolocatorCognitoProduction_sns-role",
+    "roleExternalId": "echolocatorCognitoProduction_role_external_id",
+    "policyName": "echolocatorCognitoProduction-sns-policy",
+    "smsAuthenticationMessage": "Your authentication code is {####}",
+    "smsVerificationMessage": "Your verification code is {####}",
+    "emailVerificationSubject": "Your verification code",
+    "emailVerificationMessage": "Your verification code is {####}",
+    "defaultPasswordPolicy": false,
+    "passwordPolicyMinLength": 8,
+    "passwordPolicyCharacters": [
+        "Requires Lowercase",
+        "Requires Uppercase",
+        "Requires Numbers",
+        "Requires Symbols"
+    ],
+    "requiredAttributes": [
+        "email"
+    ],
+    "userpoolClientName": "echolocatorCognitoProduction_app_client",
+    "userpoolClientGenerateSecret": true,
+    "userpoolClientRefreshTokenValidity": "90",
+    "userpoolClientReadAttributes": [
+        "email",
+        "phone_number",
+        "updated_at",
+        "email_verified",
```

Comment:
This ended up being the culprit -- Cognito wasn't allowing end users to read the `email_verified` attribute.

Reply:
Curiously, this was set to only "email" for both of the previous configurations. Maybe something changed with the CloudFormation rules for this service.

Reply:
That's a good point. It's possible that we adjusted something out of band without knowing it, too.